bin/cli.js

```diff
@@ -186,7 +186,9 @@ function invoke(env) {
   commander.parse(process.argv);
 
   Promise.resolve(pending).then(function() {
-    commander.help();
+    commander.outputHelp()
+  }).then(function () {
```

Comment:
`.then` here doesn't really add anything. it would be the same / better to put exit in the same block with help printing.